EIPS/eip-3473.md

```diff
@@ -0,0 +1,463 @@
+---
+eip: 3473
+title: ERC-659 Multiple Callable Bonds Standard
+author: Yohji Sakamoto <info@sgm.finance>
+discussions-to: https://github.com/ethereum/EIPs/issues/3467
+type: Standards Track
+category: ERC
+status: Draft
+created: 2021-04-05
+requires: 20
+---
+
+## Simple Summary
+
+A standard interface for contract, that manage multiple callable bonds. A single contract includes any given number of bond classes, bond nonce, bond balance of an address. This standard provides independent functions to read, transfer any collection of bonds, as well as allow bonds to be redeemed from the bond issuer if certain conditions are met. This token standard can replace current ERC20 LP token. ERC-659 has more complex data structure, which will allow the LP token to store more information, and allow the developer to build more sophisticated logic for the redemption and reward system of the DEFI project in question.
+
+![img](https://github.com/sgmfinance/erc-659/blob/main/blob/master/BOND.png?raw=true) 

```

Comment:
Please bring this image into the EIPs repo. Refer to EIP-1 to see how where it should be place in the repo.